overlord/fdestate/fdemgr.go

```diff
@@ -61,9 +89,43 @@ func (m *FDEManager) Ensure() error {
 
 // StartUp implements StateStarterUp.Startup
 func (m *FDEManager) StartUp() error {
+	if m.preseed {
+		// nothing to do in preseeding mode
+		return nil
```

Comment:
don't we need to track that the Manager wasn't initialized?

Reply:
I thought about having an explicit check for preseeding and returning an error in relevant API calls, like we sometimes do in devicestate. However, we don't have any APIs as of now yet

Reply:
Let me know if this is fine, or whether you prefer to perhaps set an initErr like we do for failed state initialization already with this branch.

Reply:
Maybe you can set initErr to something mentioned we are in preeesed?